src_mod/core/entity.h

```diff
@@ -26,7 +26,7 @@
 
 #include "q_math.h"
 #include "q_shared.h"
-#include <game/surfaceflags.h>
+#include "../game/surfaceflags.h"
```

Comment:
Revert it back.
Compiler uses -I flag which sets "src_mod" dir as include directory.